saleor/graphql/stock/schema.py

```diff
@@ -0,0 +1,45 @@
+import graphene
+import graphene_django_optimizer as gql_optimizer
+
+from ...stock import models
+from ..core.fields import FilterInputConnectionField
+from ..decorators import permission_required
+from .filters import StockFilterInput
+from .mutations import StockBulkDelete, StockCreate, StockDelete, StockUpdate
+from .types import Stock
+
+
+class StockQueries(graphene.ObjectType):
+    stock = graphene.Field(
+        Stock,
+        description="Look up a stock by ID",
+        id=graphene.ID(required=True, description="ID of an warehouse"),
+    )
+    stocks = FilterInputConnectionField(
+        Stock,
+        description="List of stocks.",
+        filter=StockFilterInput(),
+        query=graphene.String(),
+    )
+
+    @permission_required("stock.manage_stocks")
+    def resolve_stock(self, info, **kwargs):
+        stock_pk = kwargs.get("id")
```

Comment:
Is this Base64 ID or pk -primary key? If the former then I would name it `stock_id`. We use both across our API so it's good to have a clear distinction in naming.